### ISSUE TITLE
Plan S implementation

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1370]
Plan S is a new European-led initiative to force open access publishing of research. It seems like the most likely outcome will be a massive disruption of current scholarly publishing models (https://en.wikipedia.org/wiki/Plan_S). It is not final yet, but I believe we should start making ourselves as compliant as possible. Full implementation details here: https://www.coalition-s.org/principles-and-implementation/

Below is a checklist of conditions and whether or not I think we are already compliant.

---

**Basic mandatory conditions for all publication venues:**

- [x] cOAlition S emphasises the need for high quality journals, therefore requiring journals/platforms to have a solid system in place for review according to the standards within the relevant discipline and guided by the core practices and policies outlined by the Committee on Publication Ethics (COPE). Details must be openly available on the respective journal and platform websites. In particular, payment of publication fees or waiver status must not in any way influence the editorial decision-making process on the acceptance of a paper.
- [x] The journal/platform must provide, on its website, a detailed description of its editorial policies and decision-making processes.
- [ ] In addition, at least basic statistics must be published annually [by the journal/platform], covering in particular the number of submissions, the number of reviews requested, the number of reviews received, the approval rate, and the average time between submission and publication.
- [x] The journal/platform must accept the retention of copyright by the authors or their institutions, at no extra cost. Licenses to publish must preserve the right and responsibility of the author/institution to make the VoR or the AAM of the article Open Access immediately upon publication, under an open license as defined in Part II Section 2 above.
- [x] The journal/platform must either enable authors to publish with immediate and permanent Open Access (without any kind of technical or other form of obstacles) under an open license as defined in Part II Section 2 above, or to deposit the AAM or VoR in an Open Access repository at no extra cost and under an open license as defined in Part II Section 2 above. In either case, no embargo period can be applied (including for early view versions, i.e., online VoR before inclusion in an issue).

**Mandatory technical conditions for all publication venues:**

- [x] Use of persistent identifiers (PIDs) for scholarly publications (with versioning, for example, in case of revisions), such as DOI (preferable), URN, or Handle.
- [x] Deposition of content with a long-term digital preservation or archiving programme (such as CLOCKSS, Portico, or equivalent). [CLOCKSS costs $242 pa; portico costs $250 pa -ac]
	- @drjwbaker update 02/07/20: I think we do this with Zenodo + UK Web Archive.
- [x] High-quality article level metadata in standard interoperable non-proprietary format, under a CC0 public domain dedication. Metadata must include complete and reliable information on funding provided by cOAlition S funders (including as a minimum the name of the funder and the grant number/identifier).
- [x] Machine-readable information on the Open Access status and the license embedded in the article, in standard non-proprietary format.

**Strongly recommended additional criteria for all publication venues:**

- [x] Support for PIDs for authors (e.g., ORCID), funders, funding programmes and grants, institutions, and other relevant entities.
- [x] Registering the self-archiving policy of the venue in SHERPA/RoMEO. [Done #1794]
- [ ] Availability for download of full text for all publications (including supplementary text and data) in a machine-readable community standard format such as JATS XML. [this is an XML format  designed for journal articles. They want us to publish in XML]
- [x] Direct deposition of publications (in a machine-readable community standard format such as JATS XML, and including complete metadata as described above) by the publisher into author designated or centralised Open Access repositories that fulfil the Plan S criteria.
- [ ] OpenAIRE compliance of the metadata. [this is  an XML scheme for declaring metadata about articles. Including information on embargo periods and such]
- [x] Linking to data, code, and other research outputs that underlie the publication and are available in external repositories.
- [ ] Openly accessible data on citations according to the standards by the Initiative for Open Citations (I4OC). [this involves  signing up for CrossRef  (free) and sending them xml-formatted i nformation about citations in our papers. In return we can get back references about our papers.]

**1.2 Specific conditions applicable to Open Access journals and Open Access publishing platforms:**

- [x] The journal/platform must be registered in the Directory of Open Access Journals (DOAJ) or in the process of being registered.

**In addition, the following criteria are required:**

- [x] Open Access journals must not have a mirror/sister subscription journal with substantial overlap in editorial board to avoid business models charging for both access and publication. Such journals will de facto be considered ‘hybrid’ journals.
- [x] Transparent costing and pricing: information on the publishing costs and on any other factors impacting the publication fees must be openly available on the journal website/publishing platform (see also Part II Section 5 above).
- [x] The journal/platform must provide APC waivers for authors from low-income economies and discounts for authors from lower middle-income economies, as well as waivers and discounts for other authors with demonstrable needs. Waiver policies must be described clearly on the journal website/platform and statistics on waivers requested and granted must be provided.

*@drjwbaker update 02/07/20: confirm per https://github.com/programminghistorian/jekyll/issues/1370#issuecomment-652860793 that we don't need to comply with Section 2 as we are not an Open Access Repository. Rather, we should - as good practice - deposit with an Open Access Repository that is compliant and therefore ensures long term preservation of the journal. Our existing arrangements - Zenodo + UK Web Archive - are just that.*

**~~2. Requirements for Open Access Repositories~~**

**~~2.1 Requirements for Open Access repositories:~~**

- [ ] ~~The repository must be registered in the Directory of Open Access Repositories (OpenDOAR) or in the process of being registered.~~

**~~In addition, the following criteria for repositories apply:~~**

**~~Mandatory criteria for repositories:~~**

- [x] ~~Use of PIDs for the deposited versions of the publications (with versioning, for example in case of revisions), such as DOI (preferable), URN, or Handle.~~
- [x] ~~High quality article level metadata in standard interoperable non-proprietary format, under a CC0 public domain dedication. This must include information on the DOI (or other PIDs) both of the original publication and the deposited version, on the version deposited (AAM/VoR), and on the Open Access status and the license of the deposited version. Metadata must include complete and reliable information on funding provided by cOAlition S funders (including as a minimum the name of the funder and the grant number/identifier).~~
- [x] ~~Machine readable information on the Open Access status and the license embedded in the article, in standard non-proprietary format.~~
- [x] ~~Continuous availability (uptime at least 99.7%, not taking into account scheduled downtime for maintenance or upgrades)~~.
- [ ] ~~Helpdesk: as a minimum an email address (functional mailbox) has to be provided; a response time of no more than one business day must be ensured.~~

**~~Strongly recommended additional criteria for repositories:~~**

- [x] ~~Manuscript submission system that supports both individual author uploads and bulk uploads of manuscripts (AAM or VoR) by publishers.~~
- [ ] ~~Full text stored in a machine-readable community standard format such as JATS XML.~~
- [x] ~~Support for PIDs for authors (e.g., ORCID), funders, funding programmes and grants, institutions, and other relevant entities.~~
- [ ] ~~Openly accessible data on citations according to the standards by the Initiative for Open Citations (I4OC).~~
- [x] ~~Open API to allow others (including machines) to access the content. A compliant API must be free to access without any barrier. A light authentication mechanism such as a token for ‘power users’ – e.g., high-traffic collaborators – is acceptable as long as there is a totally open/anonymous route too.~~
- [ ] ~~OpenAIRE compliance of the metadata.~~
- [ ] ~~Quality assurance processes to link full-text deposits with authoritative bibliographic metadata from third party systems, e.g., PubMed, Crossref, or SCOPUS where feasible.~~

**Comment by acrymble** [comment]
This seems like a primarily technical team task. Can I pass this to @mdlincoln for consideration and comment?

**Comment by mdlincoln** [comment]
- We already deposit content to Zenodo for long term archiving
- Can definitely generate further metadata on the article as well as the license on each page
- DOIs/URNs will require some institutional partnership, such as with a university library that has a contract to a DOI provider and can register the DOIs for us. Given that the incorporation process is going on in the UK, I think this makes sense for it to be a UK institution.
- The technical challenge on this one is providing versioned and *rendered* views of our lessons. I can generate a list of lesson versions based on commit history, queried via the GitHub API. But those commits will point only to to the markdown file, and GitHub does not render out earlier versions of the HTML. I believe this is sufficient for our purposes, provided that we've a DOI provider in place. But absent that, we'd need a totally different build process that would have to happen outside of GitHub.

**Comment by acrymble** [comment]
@mdlincoln can I present this at your next technical meeting? There's a big opportunity for us to be world leading on the technical end of publishing so I think this is important, and I'd value the chance to discuss it with your team.

**Comment by acrymble** [comment]
The most difficult of these to implement is probably:

> Helpdesk: as a minimum an email address (functional mailbox) has to be provided; a response time of no more than one business day must be ensured.

The timeline on that is just unrealistic for an organization like us. I've just spoken with the President of the Royal Historial Society in the UK, and she confirms this is a key concern for others too, so may well be challenged. Apparenty the Bodleian Library at Oxford said they wouldn't be able to guarantee this 24 hour turnaround. So if they can't, it's not realistic for us to either.

**Comment by drjwbaker** [comment]
We also don't currently publish our shared gmail. I monitor it because it is where we get the Ltd registered address, but if we publish it that might lead to an increase in messages to triage.

**Comment by acrymble** [comment]
The Royal Historical Society has just published a report and recommendations on Plan S.  I wasn't aware that the full set of recommendations may not be adopted. At least not in all countries. We should seek financial support to implement the new changes, since we're a small publisher and not one of the big boys.

https://royalhistsoc.org/policy/publication-open-access/plan-s-and-history-journals/

**Comment by drjwbaker** [comment]
Thanks for keeping on top of this @acrymble. Of the things you posted at the top of this ticket, on your reading of the RHS report do any look like falling away? On £, let's keep an eye out for that.

**Comment by acrymble** [comment]
I don't know @drjwbaker. I raised some concerns to the RHS about DOAJ's discretion and lack of responsiveness, and of the cost of LOCKKS, which assumes you have money to pay for it.

**Comment by mdlincoln** [comment]
I'm scheduling a meeting with our scholarly communications librarian, who has more experience than I in the world of DOI vendors, to get a better picture (and budget) for what a DOI system for lessons (and lesson revisions) might entail.

**Comment by mdlincoln** [comment]
Outcomes of my discussion:

## Cost

The big providers are DataCite and CrossRef. [DataCite's heavy annual membership/cheap DOI model](https://datacite.org/pricelist.html) is very much designed for large repositories with thousands of objects to register. If we were to directly engage a provider, I'd instead recommend [CrossRef](https://www.crossref.org/fees/#content-registration-fees) which has a $275 annual fee + a one-time $1 fee per DOI we register.

However, there is also the option of trying to attach ourselves to a university publisher that already has a membership with one of these providers, and who would be willing to give us DOIs instead for free or very minor charge. This is a possibility I could try to bring up with CMU, however given the incorporation in the UK and I'm but a staff member, not faculty, I'm not sure if that's the best way forward. I can float the idea though.

## Mechanics

Providers have a web interface where we would **reserve** a DOI once a lesson is accepted for publication. Editors would have to enter the DOI into a YAML field for us to properly display it online. Once the lesson goes live, we go back to the provider interface and **register** the final public URL with the provider along with metadata like authors etc.

## What do we put the DOIs on? Versioning?

I asked specifically about our most common kinds of edits: typos and broken links. The community practice seems to be that people are fine with those not needing to count as "new versions" of an article. We would just update the site, and anyone with the DOI gets the latest content.

Trying to handle more substantive changes to lesson content as new versions displaying at the same URL would be very painful both given the current way that providers handle versioned DOIs, as well as the way our platform works. Although we have a full git history for all our files, we only have one _rendered_ version of the site. We cannot point people to an earlier HTML version, only to an earlier Markdown version.

However, I believe this is trying to solve the wrong problem. **My recommendation*: If we are making a large enough edit to a lesson that we believe it crosses the threshold of a "new version" that a reasonable reader would want to uniquely distinguish from an older one, then we shouldn't be trying to put that content at the original URL. I would argue that we should actually be creating a new page, not editing an existing one. In that way, we sidestep the infrastructural headaches, but more importantly present a a much clearer set of patterns for our readers:

1. The original lesson stays up, rendered in HTML at the old url it's always been at. 
2. The new version has a new URL and a new DOI. New means new!
3. Maybe we add a manual note on old lesson pointing to the new one (which is what we would do if we were choosing to retire the original lesson.) 

Whether or not such a revision ALSO triggers a new peer review round is a question worthy of discussion and documentation by the editorial team.

**Comment by drjwbaker** [comment]
@mdlincoln Thanks for putting the hard work into this. It raises an important issue around how we present substantial revisions, and your suggested new workflow seems astute to me, both in terms of infrastructure and publishing: for whilst we do have a modified date, we currently don't have an easy to understand record of what changed. This proposal solves that.

On the CrossREF cost, I'd like to think we could absorb that in time, once we build our financial base. But we aren't there yet. And any institutional support/help we can find through affiliation would be welcome. I don't think the UK incorporation should get in the way of that affiliation coming from a non-UK entity.

**Comment by mdlincoln** [comment]
>whilst we do have a modified date, we currently don't have an easy to understand record of what changed. This proposal solves that.

I think another component to this is better documenting for ourselves, *and then documenting for our authors &readers*, what we consider minor revisions that do not constitute creating a new "version" (fixing broken links, typos in code, incorrectly-uploaded datasets) and thus don't get big signposts in our UI. Documentation of those minor changes just comes from the built-in git file history that we already link to.

**Comment by drjwbaker** [comment]
Agree. Tension between "we are transparent because of git" and the way we show our readers the important changes in that git history in forms that they can grasp.

**Comment by mdlincoln** [comment]
To be clear, I think it's ok for us to do "silent bugfixes" like we currently do on lessons, with only the commit history as log.

It's for the bigger updates that we should probably get comfortable with just creating a new file and DOI.

**Comment by mdlincoln** [comment]
I've talked some more with CMU colleagues and there is interest in continuing institutional support of the project in many ways, one perhaps being DOIs from our library. I will be able to follow up on this after the winter break.

**Comment by drjwbaker** [comment]
@mdlincoln I have an update on this that I'd like to keep to email for now (see my email today!)

**Comment by mdlincoln** [comment]
Perfect; responded.

**Comment by drjwbaker** [comment]
Quick update/placeholder: we are looking at Feb for a decision on whether or not the proposed DOI infrastructure support comes through. I'll keep you posted when I know more.

**Comment by mdlincoln** [comment]
Please keep me in the loop particularly as questions about exact integrations with their provider come up (we don't have a server so we can't do calls to an API; need a manual option for reserving & registering, etc.) I'm glad to join in correspondence or conference calls as is useful.

**Comment by drjwbaker** [comment]
Of course. Early-Feb will be about establishing if we get the support. After that we need to figure out implementation. My plan is that - initially - me and you coordinate and participate in discussion. We can then decide how to delegate work from there.

**Comment by drjwbaker** [comment]
Quick update. Our proposed partner on the DOI thing are meeting on 10 Feb (I'm not in the meeting) to discuss internal processes for minting our DOIs. I expect that a) this will green light the support and b) will generate some questions for us. I'll do triage on the latter when I have them.

**Comment by drjwbaker** [comment]
Quick update: our proposed partner is checking with CrossREF if they can mint DOIs for us under their current membership, or if - because we are an non-profit org - we need our own membership to mint DOIs. Whatever the result, our proposed partner will support this (e.g. if needs be, pay for an manage our CrossREF membership, pay per DOI minted, et cetera)

**Comment by acrymble** [comment]
There are a lot of steps to this ticket. Would it help if we broke it up into smaller tickets surrounding a milestone? I don't think it's easy to know what needs to be done?

I'd also suggest this might be the type of task we want to pay someone on the team to do, as it's quite complex.

**Comment by mdlincoln** [comment]
I will open a ticket shortly, specifically regarding DOI policy - I have some larger thoughts on it that need a dedicated thread.

**Comment by acrymble** [comment]
I've started lobbying the Royal Historical Society for support for this work, as there don't seem to be any grants to cover it. Someone on twitter also suggested we approach the Marc Fitch Fund.

**Comment by mdlincoln** [comment]
What parts are the ones that require funding?

**Comment by acrymble** [comment]
@mdlincoln any of it could benefit from being able to hire someone to dedicate attention to it. It's a big task. And one brought on by external forces.

**Comment by acrymble** [comment]
It's worth stating that these Plan S guidelines are not yet set in stone, and a public consultation is currently open that we should consider expressing opinions on, as the outcome could result in extra effort required from us (https://www.ukri.org/funding/information-for-award-holders/open-access/open-access-review/)

**Comment by drjwbaker** [comment]
@acrymble I've opened a fresh ticket re Plan S and the URKI consultation.

**Comment by mdlincoln** [comment]
We've already implemented embedded article-level metadata in `<meta>` tags for each lesson, and XML will be generated as part of the crossref deposit to create DOIs, so most of the technical aspects of this are finished or, with DOIs, waiting on Sussex.

**Comment by acrymble** [comment]
In order to conclude this (very complicated and complex series of recommendations) in this ticket, I propose we categorise the outstanding items into:

- Wont do
- Can't afford to do without funding
- Will do

I propose the following breakdown of those (but welcome in put from anyone interested):

### Won't Do

- The journal/platform must provide, on its website, a detailed description of its editorial policies and decision-making processes. In addition, **at least basic statistics must be published annually, covering in particular the number of submissions, the number of reviews requested, the number of reviews received, the approval rate, and the average time between submission and publication.** [The bold items are an unnecessary and unreasonable burden on our editors, and of unclear value to our communities. Our system does not make this easy to do automatically -ac]

- Deposition of content with a long-term digital preservation or archiving programme (such as CLOCKSS, Portico, or equivalent). [The cost of this ranges between $242 and $250 p.a. Unless someone wants to pay for this for us we will continue using Zenodo for free -ac]

- Registering the self-archiving policy of the venue in SHERPA/RoMEO. [We tried to reach out to this organization in October 2019 but were ignored -ac]

- Availability for download of full text for all publications (including supplementary text and data) in a machine-readable community **standard format such as JATS XML**. [Our material can be downloaded in a machine-readable community standard known as Markdown or HTML -ac]

- The repository must be registered in the Directory of Open Access Repositories (OpenDOAR) or in the process of being registered [We tried to do this but were refused for our Spanish publication (despite being listed for our English publication. Attempts to contact the team were ignored -ac]

- Helpdesk: as a minimum an email address (functional mailbox) has to be provided; a response time of no more than one business day must be ensured [We aim to provide a speedy response, but cannot meet this unreasonable one-day turnaround -ac]

### Can't afford to do without funding

- OpenAIRE compliance of the metadata. [We do not have in-house expertise for this requirement and unless we can get access to that expertise and funding to implement this standard, we will be unable to comply -ac]

- Quality assurance processes to link full-text deposits with authoritative bibliographic metadata from third party systems, e.g., PubMed, Crossref, or SCOPUS where feasible. [Without a product to integrate this and funding to hire someone to do it, we'll be unable to comply -ac]

### Will do

Openly accessible data on citations according to the standards by the Initiative for Open Citations (I4OC). [@mdlincoln I think this is happening already as part of the work on DOIs, yes? If not we should put this in the 'can't afford' category. -ac]


--

If we're happy with that, I suggest we document these decisions via a public blog post or submit an article to engage with the wider Open Publishing conversations and the strains they put on small publishers like us.

@mdlincoln you've put the most work into this complex ticket. Can you let me know your views on that proposal?

**Comment by drjwbaker** [comment]
> Registering the self-archiving policy of the venue in SHERPA/RoMEO. [We tried to reach out to this organization in October 2019 but were ignored -ac]

The EN edition is on the old Sherpa/Romeo via our DOAJ listing http://sherpa.ac.uk/romeo/search.php I suspect the email how caught in the shift of Sherpa/Romeo to a Jisc infrastructure (in beta, launches in July) https://v2.sherpa.ac.uk/romeo/search.html We have friends at Jisc, so can we try resending the request to help@jisc.ac.uk to get us listed (ideally, EN/FR/ES and PO) on the new service?

**Comment by acrymble** [comment]
Good idea @drjwbaker.

On the won't do's, I'm going to action this via a blog post explaining why we can't meet the plan S requirements, and use it as a conversation starter within the wider community. Then I think we can close this for now (pending the appearance of additional financial support to help implement the rest).

**Comment by drjwbaker** [comment]
> Good idea @drjwbaker.

Whose "job" do you think it is to do this?

**Comment by acrymble** [comment]
I dont' know, but I can do it. I will start a new ticket though because it's a new issue.

**Comment by drjwbaker** [comment]
Thanks Adam.

**Comment by acrymble** [comment]
I've drafted a blog post for why we won't be implementing everything here. I'd welcome comments before I move it over to the blog and close this off.

---


### Why *Programming Historian* is not fully implementing Plan-S Recommendations

*Programming Historian* has been fully Open Access since day one. For a decade now our team has been publishing academic content and making it freely and widely available online to more than 1-million people per year. In that time, we've never charged an author, reader, or a library a penny for accessing our material.

We therefore welcomed the arrival of 'Plan-S', an open access initiative in scholarly publishing that encourages new standards across the sector. As practicing academics, we look forward to a day when knowledge is more widely available to those who can benefit from it. We're happy to report that we already meet most of the suggested requirements, including open licenses for published content, DOIs for all articles, high quality metadata, and transparent costing (free in our case). 

But as a not-for-profit open access scholarly publisher built on the backs of thousands of hours of passion and voluntary effort, we can't, and we won't implement the entire [set of recommendations made by Plan S, in their current form (mid 2020)](https://www.coalition-s.org/addendum-to-the-coalition-s-guidance-on-the-implementation-of-plan-s/principles-and-implementation/). 

This is because our business model does not rely upon article processing charges (APCs), extensive subscription fees from a scholarly society, or a home within an academic library who pays all of the bills. We're a group of scholars who believe in what we are doing. So we put our own time (and in some cases, money) into making that happen. We have a small budget [generously gleaned from members of our community](https://www.patreon.com/theprogramminghistorian) who have decided to financially back us, as well as some [wonderful library partners who provide cash or in-kind support](https://programminghistorian.org/en/support-us).

However, our budget remains extremely tight, and we owe it to our supporters to make sure every penny goes where it's needed most. We are therefore extremely dismayed that so many of the 'Plan S' recommendations or requirements cost money, without obviously adding value to our publications or our core mission of sharing knowledge openly and for free.

We have therefore decided as a team, NOT to implement the following Plan-S requirements without direct on going financial support to do so:

### 1) The journal/platform must provide, on its website...at least basic statistics...covering in particular the number of submissions, the number of reviews requested, the number of reviews received, the approval rate, and the average time between submission and publication.

*Programming Historian* operates principally on an [open peer review process](https://github.com/programminghistorian/ph-submissions) of which we are very proud. This review system makes most of these questions answerable to anyone curious. But our system does not easily generate these requested statistics. To provide them one of our team mates would have to dedicate voluntary hours counting and checking them, and for no obvious benefit to our authors, readers, or supporters. We will therefore not be implementing this unnecessary set of metrics.

### 2) Helpdesk: as a minimum an email address (functional mailbox) has to be provided; a response time of no more than one business day must be ensured 

Our voluntary team of academics who support our publishing activities always aim to provide a speedy response to queries, but we cannot and will not impose a one-day turnaround requirement upon them. This is unreasonable to demand of even paid employees.


### 3) Deposition of content with a long-term digital preservation or archiving programme (such as CLOCKSS, Portico, or equivalent).

We have for many years been depositing our entire publications in [Zenodo](https://zenodo.org/record/3525082) which offers free Archiving for research organizations around the world. The suggested services, CLOCKSS or Portico, cost between $242 and $250 per year, and assume a connection to a subscribing academic library or the means to pay these fees, which we do not have.


### 4) Registering the self-archiving policy of the venue in SHERPA/RoMEO. 

We tried to register our policy on self-archiving with SHERPA/RoMEO in October 2019, but our email was ignored. If the service cannot meet the demands upon it, it cannot be part of a required infrastructure.


### 5) The repository must be registered in the Directory of Open Access Repositories (OpenDOAR) or in the process of being registered 

We tried to register our Spanish and French publications but we were refused. Our English publication (which is identical in every way except for language) was accepted. All attempts to discuss this further have been ignored. If a service cannot meet the demands upon it or makes arbitrary and inconsistent decisions about listing journals, it cannot be part of a required infrastructure.


### 6) OpenAIRE compliance of the metadata

We provide extensive metadata with our publications, which include detail above and beyond most academic publications (such as reviewer and editor names). However, we do not have OpenAIRE expertise within our voluntary team, nor do we have the money to pay for training or to pay team members for their time to develop this expertise in order to comply with a new set of standards in metadata. Without direct and ongong support we will not be able to comply with this requirement.


### 7) Availability for download of full text for all publications (including supplementary text and data) in a machine-readable community standard format such as JATS XML.

Our publications are all formatted in [Markdown](https://en.wikipedia.org/wiki/Markdown), a widely used and human/machine readable format. We do not have anyone on our voluntary team with expertise in JATS XML, nor do we see the direct benefits to our authors or readers of providing our material in such a specific format. We do, however, have a lesson published that anyone can use to [learn how to convert documents between a range of file formats](https://programminghistorian.org/en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown) using Pandoc (with thanks to Dennis Tenen and Grant Wythoff), and we would encourage anyone datamining to build their own skills rather than expecting publishers to do their data cleaning work for them.


---

We remain enthusiastic about Open Access and conversations about how we can all work better together. But we aren't Elsevier or Taylor and Francis. We don't collect APCs or subscription fees from libraries. We haven't got 1000% profit margins. We're a group of scholars trying to improve our field by doing things well, efficiently, and inexpensively. 

Our time is overwhelmingly voluntary, and we want to use this blog post to remind policy makers operating in the Open Access space that we're here too, we've done a great job of open access for years now even without your prompting, and we'd like a seat at the table. Because the decisions you make have serious implications for teams like ours, and we'd like to be here for the long-term because we've got a lot more to offer.

**Comment by mdlincoln** [comment]
I don't necessarily disagree with the statements in the blog post, Adam.

But it reads very much as a policy statement / screed by the entire Programming Historian team, and that amounts to a significant policy decision that falls under our quorum guidelines, significant enough to make sure to email the listserv and make sure that there's general assent to publishing this.

**Comment by acrymble** [comment]
I've emailed this to the team for comment. If I don't hear any serious concerns or objections by 29 June 2020 I'll post it. Otherwise it will be put on the next meeting agenda.

**Comment by acrymble** [comment]
Just a reminder to the @programminghistorian/project-team that there is about a week left to comment on this before I post it.

**Comment by drjwbaker** [comment]
On reflection based on reaction to https://programminghistorian.org/posts/plan-s, I don't know that we need to comply with the "Requirements for Open Access Repositories" section, but I do think we need to have a workflow for depositing *with* a compliant Open Access Repository. I'll add this to my list and when I get some time, try to work through the detail. I do have some expertise here (I used to manage an Institutional Repository) so I think I can come up with a viable proposal. Apologies for not chipping in soon. This fell down my priority list.

**Comment by drjwbaker** [comment]
Note for later: our archive is publicly accessible at https://www.webarchive.org.uk/wayback/en/archive/*/http://programminghistorian.org/

**Comment by drjwbaker** [comment]
Adding some intel from Johan Rooryck at cOAlition S:

- Helpdesk: "this is not applicable to journals, it only applies to repositories". This confirms my feeling that we can safely ignore section 2. 
- Long term preservation: this is a requirement, but I think we already comply with Zenodo + UK Web Archive.
- SHERPA/RoMEO: "This is only strongly recommended, not a requirement". Note for https://github.com/programminghistorian/jekyll/issues/1794
- OpenAIRE compliance of the metadata: "This is only strongly recommended, but not required"
- JATS XML: [Amelica-Redalyc](http://amelica.org/index.php/en/home/) might be able to support.

I know we've already gone through a will/won't processs https://github.com/programminghistorian/jekyll/issues/1370#issuecomment-627208945, but given the response to the blog post, I'm going to cover this off next week https://github.com/programminghistorian/jekyll/issues/1370#issuecomment-652260114, as I think a) we are doing more than we realise, b) there is less we need to do than we realised, and c) there is more support out there than we realised.
